js_modules/dagster-ui/packages/ui-core/src/metadata/MetadataEntry.tsx

```diff
@@ -39,6 +39,8 @@ export const HIDDEN_METADATA_ENTRY_LABELS = new Set([
   'dagster-dbt/exclude',
   'dagster_dbt/manifest',
   'dagster_dbt/dagster_dbt_translator',
+  'dagster_sling/translator',
+  'dagster_sling/replication_config',
```

Comment:
It's coming from `dagster_embedded_elt`, we should use that as the namespace.

And since that namespace is a bit vague, we should use `dagster_sling_translator` and `sling_replication_config` or something of the like